edit.php

```diff
@@ -48,6 +48,12 @@
         fb("Failure on $key => ". print_r($value, TRUE) ." for $edit_user");
       }
       fb("Success on $key => ". print_r($value, TRUE) ." for $edit_user");
+    } elseif (is_array($new_user_data[strtolower($key)])) {
```

Comment:
This `strtolower` is on the `$key` not the `$value`. The only things being lowercased are the field names. 

To your question about unicode though, I'm not sure how it interacts with unicode. I was using it to stay consistent with the way the test already happens on [line 44](https://github.com/gene1wood/phonebook/blob/582c803bced2ce40bddfab73d050a4bc72466243/edit.php#L44).